create init for tellor flex

tallywiesenberg opened the pull request:
points tellorflex to governance

can only be called once

tkernell commented on contracts/TellorFlex.sol:
```diff
@@ -71,17 +73,32 @@ contract TellorFlex {
+        );
+
+        governance = _governanceAddress;
+        initted = 1;
```
couldn't we remove `initted` and just `require(governance == address(0))` so that `init` can only be called once?

tkernell replied:
and then we'd remove the `_governance` argument from the constructor

tallywiesenberg replied:
good call, will do!

themandalore commented:
@tallywiesenberg , a whole bunch of conflicts...

tallywiesenberg commented:
- added some mock funcitons in `MockGovernance.sol` to mock governance calls to change  the `reporterLock`, `stakeAmount`, and `governanceAddress.
- updated tests to call the `init()` function in the `beforeEach`
- implemented @tkernell 's recommendation to remove the `initted` var

tallywiesenberg commented on contracts/TellorFlex.sol:
```diff
@@ -81,24 +82,72 @@ contract TellorFlex {
+     * @dev Changes governance address
+     * @param _newGovernanceAddress new governance address
+     */
+    function changeGovernanceAddress(address _newGovernanceAddress) external {
```
aha I gotta remove these functions and their tests then

tallywiesenberg commented:
Removed the deprecated functions + their tests
